ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/flow-tracking.md

```diff
@@ -93,52 +93,45 @@ Software export of IPFIX data records enabled.
 
 ```eos
 !
-flow tracking sampled
```

Comment:
The new config under "flow tracking sampled" is missing the exporters and maybe more.

Reply:
Fixed. Something went wrong while rebasing.